addons/hr_holidays/models/hr_leave.py

```diff
@@ -595,15 +595,22 @@ def name_get(self):
                 if leave.leave_type_request_unit == 'hour':
                     res.append(
                         (leave.id,
-                        _("%s on %s : %.2f hours") %
-                        (target, leave.holiday_status_id.name, leave.number_of_hours_display))
+                         _("%s on %s : %.2f hours") %
```

Comment:
We could add the date here as well.
( "%s on %s : %.2h hours on leave.date_from") ?

Reply:
J'y avais pensé avant de voir le brol que c'était:

![image](https://user-images.githubusercontent.com/4848797/73649849-ff3a7200-4680-11ea-9ca1-d1f1cebf68e5.png)


Mais si tu y tiens ;)

Reply:
ou le timeoff demo:

![image](https://user-images.githubusercontent.com/4848797/73649928-2bee8980-4681-11ea-9127-9b1fddbb2df8.png)